runtime/gc_vlhgc/CardListFlushTask.cpp

```diff
@@ -119,8 +122,16 @@ MM_CardListFlushTask::run(MM_EnvironmentBase *envBase)
 						/* For Marking purposes we do not need to track references within Collection Set */
 						MM_HeapRegionDescriptorVLHGC *referencingRegion = interRegionRememberedSet->tableDescriptorForRememberedSetCard(card);
 						if (referencingRegion->containsObjects() && !referencingRegion->_markData._shouldMark) {
-							Card *cardAddress = interRegionRememberedSet->rememberedSetCardToCardAddr(env, card);
-							writeFlushToCardState(cardAddress, gmpIsActive);
+							bool needToFlush = true;
+							if (needToCheckMarkMap) {
+								void* heapAddress = convertHeapAddressFromRememberedSetCard(envBase, card);
```

Comment:
interRegionRememberedSet is available. you can use its  convertHeapAddressFromRememberedSetCard(), and no need to clone it in this class